examples/scrollcontainer/scrollcontainer/app.py

```diff
@@ -17,22 +17,60 @@ def __init__(self, text):
 
 
 class ScrollContainerApp(toga.App):
+    TOGGLE_CHUNK = 10
+
     def startup(self):
+        self.top_label = toga.Label("I'm top", style=Pack(width=100))
+        top_box = toga.Box(children=[self.top_label])
+        self.scroller = toga.ScrollContainer(horizontal=False, on_scroll=self.on_scroll)
```

Comment:
This needs a `flex=1` style, otherwise the widget will have default height. If that isn't happening on Windows, that's a bug.